_data/toc.yaml

```diff
@@ -1027,18 +975,87 @@ manuals:
       title: Using Docker Cloud and Packet.net
     - path: /docker-cloud/infrastructure/cloud-on-aws-faq/
       title: Using Docker Cloud on AWS
-  - title: Cloud API reference
-    path: /apidocs/docker-cloud/
-  - title: Cloud stack file YAML reference
-    path: /docker-cloud/apps/stack-yaml-reference/
+  - sectiontitle: Standard Node and App Management
+    section:
+    - path: /docker-cloud/standard/
+      title: Standard Cloud Node and App Management
+    - sectiontitle: Getting Started
+      section:
+      - path: /docker-cloud/getting-started/
+        title: Getting started with Docker Cloud
+      - path: /docker-cloud/getting-started/intro_cloud/
+        title: Introducing Docker Cloud
+      - path: /docker-cloud/getting-started/connect-infra/
+        title: Link to your Infrastructure
+      - path: /docker-cloud/getting-started/your_first_node/
+        title: Deploy your first node
+      - path: /docker-cloud/getting-started/your_first_service/
+        title: Deploy your first service
+      - sectiontitle: Deploy an application
+        section:
+        - path: /docker-cloud/getting-started/deploy-app/1_introduction/
+          title: Introduction to Deploying an app in Docker Cloud
+        - path: /docker-cloud/getting-started/deploy-app/2_set_up/
+          title: Set up your environment
+        - path: /docker-cloud/getting-started/deploy-app/3_prepare_the_app/
+          title: Prepare the application
+        - path: /docker-cloud/getting-started/deploy-app/4_push_to_cloud_registry/
+          title: Push the image to Docker Cloud's Registry
+        - path: /docker-cloud/getting-started/deploy-app/5_deploy_the_app_as_a_service/
+          title: Deploy the app as a Docker Cloud service
+        - path: /docker-cloud/getting-started/deploy-app/6_define_environment_variables/
+          title: Define environment variables
+        - path: /docker-cloud/getting-started/deploy-app/7_scale_the_service/
+          title: Scale the service
+        - path: /docker-cloud/getting-started/deploy-app/8_view_logs/
+          title: View service logs
+        - path: /docker-cloud/getting-started/deploy-app/9_load-balance_the_service/
+          title: Load-balance the service
+        - path: /docker-cloud/getting-started/deploy-app/10_provision_a_data_backend_for_your_service/
+          title: Provision a data backend for the service
+        - path: /docker-cloud/getting-started/deploy-app/11_service_stacks/
+          title: Stackfiles for your service
+        - path: /docker-cloud/getting-started/deploy-app/12_data_management_with_volumes/
+          title: Data management with Volumes
+    - sectiontitle: Manage Applications
```

Comment:
Alternatively, we could change the section name to "Manage Infrastructure (Standard Mode)" and move it below "Manage Swarms (Beta Swarm Mode)".

The current "Manage Infrastructure" section is highly specific to the standard mode functionality and most of the content doesn't apply to Swarm mode.